sematic/ui/src/types/Types.tsx

```diff
@@ -693,15 +694,18 @@ function LinkValueView(props: ValueViewProps) {
 }
 
 function DatetimeValueView(props: ValueViewProps) {
-  const {valueSummary} = props;
-  const date = parseISO(valueSummary)
+  const { valueSummary } = props;
+  const date = parseISO(valueSummary);
 
   if (!valueSummary || !isValid(date)) {
-      return <Alert severity="error">Incorrect date value.</Alert>;
+    return <Alert severity="error">Incorrect date value.</Alert>;
   }
-  return (
-      <Typography>{format(date, "LLLL d, yyyy h:mm:ss a xxx", )}</Typography>
-  );
+  return <Typography>{format(date, "LLLL d, yyyy h:mm:ss a xxx")}</Typography>;
+}
+
+function ChipValueView(props: ValueViewProps) {
+  const { valueSummary } = props;
+  return <Chip label={valueSummary} variant="outlined" />;
```

Comment:
This looks no different than other types that use `<chip />`. To more explicitly depict an enum type, can you help change it to:

```TypeScript
<Chip icon={<Tooltip title="enum">
  <ExplicitOutlined style={{color: theme.palette.warning.main }}/></Tooltip>} label={valueSummary} variant="outlined" />;
```

Then it is going to look like this:
<img width="391" alt="image" src="https://user-images.githubusercontent.com/1046489/204278295-0e2fd8c5-1282-44fd-b9e5-08956cfe4747.png">


`theme` can be found with `useTheme`. There are existing examples in the code base.

Reply:
Actually @chance-an what is the value of having the `E` in the chip? I think the chip alone is sufficient.

Ideally as you mentioned offline, we would show on hover the other possible values, but we need to change the type serialization for that.

Reply:
@chance-an has already created #356 to pave the way for the proposed second iteration.

Reply:
@neutralino1 I proposed this because I want to make it more evident that this chip is from an enum type. Otherwise, it looks the same as Boolean and None types, which also use `<chip />`. So the short `E` stands for `enums`, in my proposed implementation, there is also a hover effect that tells the users that` E` stands for `enums`.

The above is an explanation of why I proposed that way. I don't feel strongly though. Improving this UI in the next iteration when we store other possible enum values is fine too.